lib/database/game.php

```diff
@@ -383,10 +354,6 @@ function getGamesListByDev($dev, $consoleID, &$dataOut, $sortBy, $ticketsFlag =
     }
 
     switch ($sortBy) {
```

Comment:
Transforming `switch` to `match` was mostly done by Rector. 

It will however not do that if cases contain complex logic (in this case `if/else`), `match` doesn't support that (which is good as prevents complexity within cases).

Rephrased it to ternary operators and Rector converted it to `match`. (Did that somewhere else, too, iirc)
Additionally, added case `1`, `$sortBy ??= 1;`, and removed the default - similar to the other match statement above.

Reply:
See https://github.com/RetroAchievements/RAWeb/pull/911#discussion_r847636111